app/src/main/java/com/example/garden_planner/GardenMethodHelper.java

```diff
@@ -169,7 +170,7 @@ public void done(List<PlantInBed> plantsInBed, ParseException e) {
 
     }
 
-    public static List<Plant> queryPlants(){
+    public static void queryPlants(String plantName, PlantAdditionAdapter adapter, List<Plant> plants){
```

Comment:
Can you explain why we're doing this?

Reply:
that was actually supposed to be deleted - I was having a lot of issues with retrieving parse objects, and this method was one of my attempts to get it to work.